src/main/java/roomescape/persistence/JdbcReservationDao.java

```diff
@@ -0,0 +1,85 @@
+package roomescape.persistence;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.Reservation;
+import roomescape.domain.ReservationTime;
+
+@Repository
+public class JdbcReservationDao implements ReservationDao {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    public JdbcReservationDao(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public List<Reservation> findAll() {
+        return jdbcTemplate.query("SELECT \n"
+                        + "    r.id as reservation_id, \n"
+                        + "    r.name, \n"
+                        + "    r.date, \n"
+                        + "    t.id as time_id, \n"
+                        + "    t.start_at as time_value \n"
+                        + "FROM reservation as r \n"
+                        + "inner join reservation_time as t \n"
+                        + "on r.time_id = t.id",
```

Comment:
```suggestion
        return jdbcTemplate.query("""
                        SELECT 
                            r.id as reservation_id, 
                            r.name, 
                            r.date, 
                            t.id as time_id, 
                            t.start_at as time_value 
                        FROM reservation as r 
                        inner join reservation_time as t 
                        on r.time_id = t.id
                        """,
```

텍스트블록을 통해 더 가독성있게 표현할 수 있습니다 :)